tilda.appdata.xml

```diff
@@ -38,6 +38,5 @@
   </screenshot>
  </screenshots>
  <url type="homepage">https://github.com/lanoxx/tilda</url>
- <updatecontact>lanoxx_at_gmx.net</updatecontact>
- <project_group>Tilda</project_group>
```

Comment:
Actually the `project_group` tag is not invalid, but just does not make sense to use in Tilda, since Tilda is not published as part of an umbrella project such as GNOME or KDE. Therefore removal is fine, but please adjust the commit message accordingly.